### ISSUE TITLE
Threshold to determine the result

### ISSUE THREAD

**Issue by VictorWangwz** [huanghoujing/AlignedReID-Re-Production-Pytorch#55]
Hi, 
Thanks for your working. I am working with ur baseline with the pretrained model u provided and have a issue to find the threshold to determine whether the recognized person is right or not. I added re-ranking after getting the q-g-dist matrix and set the threshold as 0.1. Is there any recommand value for this threshold?
Thx!

**Comment by huanghoujing** [comment]
Thanks for your attention. I have not conducted recognition, just using it for retrieval experiments. The threshold for recognition should be based on intra-class and inter-class distances. You may have to compute some statistics on the whole val or test set to fulfill your purpose.
